cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/BulkReaderTest.java

```diff
@@ -47,6 +48,25 @@ class BulkReaderTest extends SharedClusterSparkIntegrationTestBase
     QualifiedName table2 = uniqueTestTableFullName(TEST_KEYSPACE);
     QualifiedName tableForNullStaticColumn = uniqueTestTableFullName(TEST_KEYSPACE);
 
+    @Test
+    void testDynamicSizingOption()
+    {
+        Dataset<Row> data = bulkReaderDataFrame(tableForNullStaticColumn).option("SIZING", "dynamic").load();
```

Comment:
using `tableForNullStaticColumn` here seems implies that this test somehow has something to do with the null static column table, which isn't really the case other than the `row.getString(0)` below - can we switch this to just use one of the other tables?